Library/Formula/libmill.rb

```diff
@@ -0,0 +1,29 @@
+class Libmill < Formula
+  desc "Go-style concurrency in C"
+  homepage "http://libmill.org/"
+  head "https://github.com/sustrik/libmill.git"
+  url "http://libmill.org/libmill-0.16-beta.tar.gz"
+  sha256 'c6d68d8291dadf2ee111f6ceed057f2fd267e7dc334bef8d15b6316c35b6a0ca'
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "cmake" => :build
+  env :std
+
+  def install
+    if build.head?
+      system "./autogen.sh"
+    end
+    if ENV.compiler == :clang
+      ENV.append_to_cflags "-O1"
+    end
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "check"
+    system "make", "install"
+  end
+
+  test do
```

Comment:
You can't. `buildpath` and `testpath` never exist at the same time. You'll need to write a test. We use `test do` blocks to ensure installation didn't break anything after the `make check` stage and when dependencies are updated to ensure this formula still functions with them.